CustomDynamicFont/Sources/Presentation/Modules/SelectOSViewController/SelectOSViewController.swift

```diff
@@ -6,13 +6,68 @@
 //  Copyright © 2021 Nimble. All rights reserved.
 //
 
+import RxDataSources
+import RxSwift
+import SnapKit
 import UIKit
 
 final class SelectOSViewController: UIViewController {
 
+    private let tableView = UITableView()
+    private let disposeBag = DisposeBag()
+
+    private let viewModel: SelectOSViewModelProtocol
+
+    private weak var navigator: Navigatable?
+
+    init(viewModel: SelectOSViewModelProtocol, navigator: Navigatable) {
+        self.viewModel = viewModel
+        self.navigator = navigator
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        setUpLayout()
+        setUpViews()
+        bindViewModel()
+    }
+
+    private func setUpLayout() {
+        view.addSubviews(tableView)
+
+        tableView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+    }
+
+    private func setUpViews() {
         view.backgroundColor = .white
         title = R.string.localizable.selectosNavigationTitle()
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "Cell")
+    }
+
+    private func bindViewModel() {
+
+        viewModel.output.items.drive(tableView.rx.items(cellIdentifier: "Cell")) { _, model, cell in
```

Comment:
Optional: we can specify a clearer `cellIdentifier` here instead of just "Cell" for better clarification, can be "OSVersionCell" 🙏